jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -75,6 +75,7 @@
         frequency: 'H 0 * * *'
         scan: DISABLED
         timeout: 110
+        soak-repos: '--repo=k8s.io/kubernetes --repo=k8s.io/release'
```

Comment:
I have added a single `soak-repos` var in every job now. for all other jobs, it is initialized with '--bare'
( if i defined two vars, i could not figure out what i would initialize with for the 2 vars in every job)

Also updated `jenkins/bootstrap_test.py` to ensure that `soak-repos` is defined in every job in the file.